GG/src/ListBox.cpp

```diff
@@ -2122,7 +2150,18 @@ void ListBox::ValidateStyle()
         m_style &= ~(LIST_NOSEL | LIST_SINGLESEL | LIST_QUICKSEL);
 }
 
-void ListBox::AdjustScrolls(bool adjust_for_resize)
+Pt ListBox::ClientSizeExcludingScrolls() const
+{
+    // this client area calculation disregards the thickness of scrolls
+    Pt cl_sz = (LowerRight() - Pt(X(BORDER_THICK), Y(BORDER_THICK))) -
```

Comment:
the way the lines are broken here is confusing. prefer keeping a single expression together with linebreaks after a ,